### DISCUSSION TITLE
7.5.0

### DISCUSSION THREAD

**Discussion by phili67** [phili67/ecclesiacrm#1834] [General]
new version 7.5.0<br>New version 7.5.0<br><br>New CRM<br>- Everything is rewritten for<br><br>   * AdminLTE 3.1.0<br>   * bootStrap 4.6.0<br>   * php 8.0.3<br>   * slim 4.7.1<br>   <br>- vCard export for groups, cart, person, family<br>- search menu rewritten<br>- all the api are rewritten<br>- all the v2 route too.<br><br><br>Bugs correction<br><br>- Leaflet for safari<br>- CSV and PDF stats export is now efficient<br>- better cart management<br>- system backup/restore is rewritten too<br><br><br>Inner Beauty<br><br>- AdminLTE 3<br>- Jquery 3.6.0<br>- bootStrap is in the last version : 4.6.0<br>- FullCalendar is in version 4.4.2<br>- DataTable are now in 1.10.24<br><br><br>- Propelization<br>* GroupReport.pgp is propeled<br><br><br>etc ....<br><br>Inner Coherence<br><br>See full changelog here:

<hr /><em>This discussion was created from the release <a href='https://github.com/phili67/ecclesiacrm/releases/tag/7.5.0'>7.5.0</a>.</em>

**Comment by vikingkarwur** [comment]
Hello. How to update from previous version ? Thank you!

**Reply by phili67** [reply]
Ok to upgrade from an old beta version.
The only way is to upload all the files from the rc1 archive in your ecclesiacrm folder.
After you’ve to apply the rights to all the files and the folders.
Like : chown -R www-data:www-data .
